examples/labeler/chexpert/config_pattern_tutorial.py

```diff
@@ -0,0 +1,55 @@
+import os
+import tempfile
+from bllipparser import ModelFetcher
+
+
+class WeatherConfig:
+    def __init__(
+            self,
+            parsing_model_dir: str = ModelFetcher.download_and_install_model(
```

Comment:
Same comment as somewhere down, I don't this should be done, might have side effects.